.editorconfig

```diff
@@ -12,15 +12,14 @@ indent_style = space
 tab_width = 4
 
 # New line preferences
-end_of_line = lf
```

Comment:
Why was this removed?

Reply:
Because I realized this should not be something controlled by the editor, but rather by git itself. You can read on how to configure git line endings [here](https://docs.github.com/en/github/using-git/configuring-git-to-handle-line-endings). The editor should not be enforcing line endings because this will lead to inconsistencies on Windows, where git checks out files with `crlf`, but the editor uses `lf`.